### PR TITLE
fix other module checkstyle issue

### PR DESCRIPTION
**Pull request by seagle-yuan**
fix other module checkstyle issue

### CODE REVIEW

File: hugegraph-dist/src/main/java/com/baidu/hugegraph/cmd/InitStore.java [hunk 1]
```diff
@@ -47,15 +47,15 @@ public class InitStore {
     // Less than 5000 may cause mismatch exception with Cassandra backend
     private static final long RETRY_INTERVAL = 5000;
 
-    private static final MultiValueMap exceptions = new MultiValueMap();
+    private static final MultiValueMap EXCEPTION = new MultiValueMap();
```
Comment (javeme) [inline hugegraph-dist/src/main/java/com/baidu/hugegraph/cmd/InitStore.java:50]: can keep "EXCEPTIONS" name

File: hugegraph-dist/src/main/java/com/baidu/hugegraph/cmd/StoreDumper.java [hunk 2]
```diff
@@ -51,10 +56,10 @@ public void dump(HugeType table, long offset, long limit) {
         }
         String title = String.format("Dump table %s (offset %d limit %d):",
                                      table, offset, limit);
-        System.out.println(title);
+        LOG.info("title : {}",title);
```
Comment (javeme) [inline hugegraph-dist/src/main/java/com/baidu/hugegraph/cmd/StoreDumper.java:59]: merge into line 57

File: hugegraph-dist/src/main/java/com/baidu/hugegraph/cmd/StoreDumper.java [hunk 3]
```diff
@@ -51,10 +56,10 @@ public void dump(HugeType table, long offset, long limit) {
...
         for (long i = 0; i < limit && rs.hasNext(); i++) {
             BackendEntry entry = rs.next();
-            System.out.println(entry);
+            LOG.info("entry : {}",entry);
```
Comment (javeme) [inline hugegraph-dist/src/main/java/com/baidu/hugegraph/cmd/StoreDumper.java:62]: `LOG.info("{}" entry)` is ok

File: hugegraph-dist/src/main/java/com/baidu/hugegraph/cmd/StoreDumper.java [hunk 4]
```diff
@@ -54,12 +54,12 @@ public void dump(HugeType table, long offset, long limit) {
...
-        String title = String.format("Dump table %s (offset %d limit %d):",
-                                     table, offset, limit);
-        LOG.info("title : {}",title);
+
+        LOG.info("Dump table %s (offset {} limit {}):", table, offset, limit);
+
         for (long i = 0; i < limit && rs.hasNext(); i++) {
             BackendEntry entry = rs.next();
-            LOG.info("entry : {}",entry);
+            LOG.info("{}",entry);
```
Comment (javeme) [inline hugegraph-dist/src/main/java/com/baidu/hugegraph/cmd/StoreDumper.java:62]: expect a space after ","

**Review by javeme — APPROVED** [review]

File: hugegraph-dist/src/main/java/com/baidu/hugegraph/cmd/ConfDumper.java [hunk 5]
```diff
@@ -25,26 +25,32 @@
...
 import com.baidu.hugegraph.dist.RegisterUtil;
+import com.baidu.hugegraph.server.RestServer;
 import com.baidu.hugegraph.util.E;
+import com.baidu.hugegraph.util.Log;
 
 public class ConfDumper {
 
     public static final String EOL = System.getProperty("line.separator");
 
+    private static final Logger LOG = Log.logger(RestServer.class);
```
Comment (imbajin) [inline hugegraph-dist/src/main/java/com/baidu/hugegraph/cmd/ConfDumper.java:42]: use `RestServer` name for the class's log?

File: hugegraph-dist/src/main/java/com/baidu/hugegraph/cmd/StoreDumper.java [hunk 6]
```diff
@@ -22,21 +22,26 @@
...
 import com.baidu.hugegraph.testutil.Whitebox;
 import com.baidu.hugegraph.type.HugeType;
 import com.baidu.hugegraph.util.E;
+import com.baidu.hugegraph.util.Log;
 
 public class StoreDumper {
 
     private final HugeGraph graph;
 
+    private static final Logger LOG = Log.logger(RestServer.class);
```
Comment (imbajin) [inline hugegraph-dist/src/main/java/com/baidu/hugegraph/cmd/StoreDumper.java:43]: same warning

**Review by imbajin — APPROVED** [review]

File: hugegraph-dist/src/main/java/com/baidu/hugegraph/cmd/ConfDumper.java (hunk 5 above)
Comment (seagle-yuan) [reply hugegraph-dist/src/main/java/com/baidu/hugegraph/cmd/ConfDumper.java]: chage to ConfDumper

File: hugegraph-dist/src/main/java/com/baidu/hugegraph/cmd/StoreDumper.java (hunk 6 above)
Comment (seagle-yuan) [reply hugegraph-dist/src/main/java/com/baidu/hugegraph/cmd/StoreDumper.java]: change to StoreDumper

**Review by imbajin — APPROVED** [review]

**Review by javeme — APPROVED** [review]

**Pull request merged**